Update BouncyCastle caused SMB connections to crash app

Update BouncyCastle to 1.68 (#2170) caused SMB connections to crash. Need to revert to 1.66 as https://github.com/AgNO3/jcifs-ng uses.

## Exception
* __User Action:__ UI Error
* __Request:__ Application crash
* __Version:__ 3.6.0
* __OS:__ Linux Android 10 - 29
* __Device:__ generic_x86
* __Model:__ Android SDK built for x86
* __Product:__ sdk_gphone_x86
<details><summary><b>Crash log </b></summary><p>

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:399)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
Caused by: java.lang.IllegalAccessError: Illegal class access: 'jcifs.spnego.NegTokenInit' attempting to access 'org.bouncycastle.asn1.DEROutputStream' (declaration of 'jcifs.spnego.NegTokenInit' appears in /data/app/com.amaze.filemanager.debug-3CpIQtEdleIFopoYgPHdSQ==/base.apk)
	at jcifs.spnego.NegTokenInit.toByteArray(NegTokenInit.java:158)
	at jcifs.smb.SpnegoContext.initSecContext(SpnegoContext.java:207)
	at jcifs.smb.SmbSessionImpl.createToken(SmbSessionImpl.java:656)
	at jcifs.smb.SmbSessionImpl.sessionSetupSMB2(SmbSessionImpl.java:538)
	at jcifs.smb.SmbSessionImpl.sessionSetup(SmbSessionImpl.java:483)
	at jcifs.smb.SmbSessionImpl.send(SmbSessionImpl.java:369)
	at jcifs.smb.SmbSessionImpl.send(SmbSessionImpl.java:347)
	at jcifs.smb.SmbTreeImpl.treeConnect(SmbTreeImpl.java:611)
	at jcifs.smb.SmbTreeConnection.connectTree(SmbTreeConnection.java:614)
	at jcifs.smb.SmbTreeConnection.connectHost(SmbTreeConnection.java:568)
	at jcifs.smb.SmbTreeConnection.connectHost(SmbTreeConnection.java:489)
	at jcifs.smb.SmbEnumerationUtil.doShareEnum(SmbEnumerationUtil.java:145)
	at jcifs.smb.SmbEnumerationUtil.doEnum(SmbEnumerationUtil.java:218)
	at jcifs.smb.SmbEnumerationUtil.listFiles(SmbEnumerationUtil.java:279)
	at jcifs.smb.SmbFile.listFiles(SmbFile.java:1199)
	at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.doInBackground(LoadFilesListTask.java:123)
	at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.doInBackground(LoadFilesListTask.java:67)
	at android.os.AsyncTask$3.call(AsyncTask.java:378)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 more

```
</details>
<hr>

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -103,7 +103,7 @@ ext {
     jcifsVersion = '2.1.3'
     fabSpeedDialVersion = '3.1.1'
     roomVersion = '2.2.5'
-    bouncyCastleVersion = '1.68'
+    bouncyCastleVersion = '1.66'
     awaitilityVersion = "3.1.6"
     androidXTestVersion = "1.3.0"
     androidXTestExtVersion = "1.1.2"

